bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/dto/Color.java

```diff
@@ -27,20 +26,12 @@ public class Color {
     public int green;
     public int blue;
 
-    public Color(int brightness, HSBType hsbType) {
-        this.brightness = brightness;
-        int rgb = hsbType.getRGB();
```

Comment:
Another approach would have been to replace ```hsbType.getRGB()``` by ```ColorUtil.hsbTosRGB(hsbType)``` and keep the rest of this file unchanged.

Reply:
Yes, I did that initially, but decided against it since `HSBType` is no longer useful here after deprecating that method. So I decided to grab the sRGB value in the handler code instead and provide it directly in the new constructor.